### PR TITLE
Make solidus_globalize work with current solidus

### PR DESCRIPTION
**Pull request by srna**
Fixes a few calls and translation images. Also includes Fallbacks, which js required.

### CODE REVIEW

File: app/views/spree/admin/translations/_form_fields.html.erb [hunk 1]
```diff
@@ -13,14 +13,14 @@
                 <% if locale.to_s.include?('-') %>
                   <img src="https://www.localeapp.com/assets/flags/regions/<%= locale.to_s.split('-').last %>.png">
                 <% else %>
-                  <img src="https://www.localeapp.com/assets/flags/languages/<%= locale %>.png">
+                  <img src="https://l10n.xwiki.org/xwiki/bin/download/L10N/Flags/<%= locale %>.png">
```
Comment (bofrede) [inline app/views/spree/admin/translations/_form_fields.html.erb]: I think we should remove the flags.
Flags symbolizes countries, not languages.
In any case, the images should never be loaded from an external source.

**Comment by srna** [comment]
True. I have removed the flag images.

**Comment by bofrede** [comment]
+1
The failing specs are not related to these changes.
Please merge. :-)

**Review by tvdeyen — APPROVED** [review]
Thanks for the fixes. 👍 from me

Could you please squash the first three commits into one?

**Comment by srna** [comment]
Squashed into 2 commits with meaningful names.

**Comment by srna** [comment]
Any progress? Could you please merge?

**Comment by tvdeyen** [comment]
Would love. Unfortunately we have a failing build. Mind to look into it?

**Comment by srna** [comment]
I believe the build failure is not caused by my PR

**Comment by tvdeyen** [comment]
> I believe the build failure is not caused by my PR

Specs need to be fixed first, sorry. We can't tell if your changes may broke something. So, we can only merge this if the build is fixed. Thanks again for your contribution. Please open this again after the build is fixed. Thanks.

**Pull request closed without merging**